sql/dql/basic-queries/where-5.md

```diff
@@ -0,0 +1,163 @@
+---
+author: stefkn
+
+levels:
+  - beginner
+  - basic
+  - medium
+
+type: normal
+
+category: must-know
+
+inAlgoPool: false
+
+standards:
+  sql.read-single-table.0: 10
+  sql.read-single-table.1: 10
+  sql.read-single-table.2: 10
+
+links:
+  - '[SQL Tutorial: WHERE](http://www.sql-tutorial.com/sql-where-sql-tutorial/){website}'
+  - '[Like Predicate](https://en.wikibooks.org/wiki/Structured_Query_Language/Like_Predicate){website}'
+  - '[The IN Keyword](https://en.wikibooks.org/wiki/Structured_Query_Language/Quantified_Comparison#IN){website}'
+
+aspects:
+  - workout
+
+---
+
+# WHERE  (LIKE & BEWTEEN)
+
+---
+## Content
+
+Considering this table[3].
+
+The `LIKE` statement accepts a pattern string, where wildcard characters are used to denote an acceptable pattern[1].

```

Comment:
due to how footnotes are coded, they must appear in order in text (1 followed by 2 etc.)
```suggestion
The `LIKE` statement accepts a pattern string, where wildcard characters are used to denote an acceptable pattern[2].
```